README.md

```diff
@@ -12,7 +12,7 @@ mvn clean install
 In order to run the Maven Archetype and generate a sample Jakarta EE project, please execute (please ensure you have installed a [Java SE 8+ implementation](https://adoptium.net/?variant=openjdk8) and [Maven 3+](https://maven.apache.org/download.cgi)):
 
 ```
-mvn archetype:generate -DarchetypeGroupId=org.eclipse -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=1.0
+mvn archetype:generate -DarchetypeGroupId="org.eclipse" -DarchetypeArtifactId="jakarta-starter" -DarchetypeVersion="1.0.0-SNAPSHOT"
```

Comment:
When it is released, it will be version 1.0.0

Reply:
What I am thinking is, anyone that would be reading the GitHub repo is probably interested in the development version. We should reference the current release in the GH page instead.

Hope that clarifies? If you like, I could make that explicit, but there will inevitably be version differences, including semantic/behavioral differences. A way of reconciling that is to say the current release is documented in the GH page instead of the README. We can include that once the GH page is updated.

Reply:
As discussed and agreed upon on the mailing list, I have made it clearer the GH pages refers to the current release in Maven Central vis-a-vis this README, which refers to the development branch. Please merge when sensible?

Reply:
I'll go ahead and merge it. We can chenge it again if it's necessary.